tests/acceptance/features/apiSpacesShares/createFileFolderWhenSharesSpaceExist.feature

```diff
@@ -9,7 +9,12 @@ Feature: create file or folder named similar to Shares folder
       | Alice    |
       | Brian    |
     And user "Alice" has created folder "/FOLDER"
-    And user "Alice" has shared folder "/FOLDER" with user "Brian" with permissions "read,update"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | FOLDER   |
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | Viewer   |
```

Comment:
```suggestion
      | permissionsRole | Editor   |
```
Is not `read,update` permission means Editor

Reply:
`read+update` permissions for a file will be `File Editor` role. For `Editor` role of a folder we need `read+update+create+delete` permissions.